client/src/components/Header/components/NavLinks.tsx

```diff
@@ -12,36 +21,36 @@ import {
 } from '@fortawesome/free-solid-svg-icons';
 import { Link } from '../../helpers';
 import { updateUserFlag } from '../../../redux/settings';
-import envData from '../../../../../config/env.json';
+import envData from '../../../../../config/env';
```

Comment:
```suggestion
import envData from '../../../../../config/env.json';
```
I'll create a PR to fix this, because this was a bad naming choice, but we need `env.json` here as `env` gets the `env.js` file which cannot be used in the browser.